### DISCUSSION TITLE
posts/yocto-project-on-orange-pi-1/

### DISCUSSION THREAD

**Discussion by giscus** [humminglab/blog.humminglab.io#5] [Announcements]
# posts/yocto-project-on-orange-pi-1/

이 문서는 Orange Pi Zero 보드에서 Yocto Project를 이용하여 배포본을 만들고, 개별 패키지를 관리하는 방법에 대하여 설명한다.
설명은 다음과 같이 나누어서 설명을 할 에정이다.
Yocto Project 개발하기(1) - Orange Pi 보드 빌드 Yocto Project 개발하기(2) - Custom Layer 만들기 Yocto Project 개발하기(3) - 개발 시 로컬 패키지 관리하기 Yocto Project 개발하기(4) - Yocto SDK 빌드 Yocto Project 개발하기(5) - Yocto eSDK 이용한 개발 모델 이 문서에서는 첫번째 과정인 필요한 layer를 추가해서 빌드하여 타겟에 올려서 동작을 확인하는 과정을 설명한다.

https://blog.humminglab.io/posts/yocto-project-on-orange-pi-1/

**Comment by alfonder** [comment]
The instruction doesn't fit. 
After "bitbake core-image-minimal" there are plenty of errors and the resulting image can't be built.
It's necessary to fix the document.

**Reply by aqwerf** [reply]
Thanks for your comment.
The mistake I made was the directory where I would run the "source ...." script.
I updated it. 

- bad
```shell
$ source ./oe-init-build-env
```
- good 
```shell
$ cd ~/yocto
$ source poky/oe-init-build-env
```
